src/main/java/nextstep/fp/Lambda.java

```diff
@@ -26,31 +27,25 @@ public void run() {
         }).start();
     }
 
-    public static int sumAll(List<Integer> numbers) {
+    public static int sumConditional(List<Integer> numbers, Conditional c) {
         int total = 0;
         for (int number : numbers) {
-            total += number;
+            if (c.test(number)) {
+                total += number;
+            }
         }
         return total;
     }
 
+    public static int sumAll(List<Integer> numbers) {
+        return sumConditional(numbers, (number) -> true);
```

Comment:
```suggestion
        return sumConditional(numbers, number -> true);
```

괄호는 굳이 작성해주지 않으셔도 됩니다!

Reply:
넹!!